sbt-plugin/src/main/scala/play/soap/sbtplugin/SbtWsdl.scala

```diff
@@ -142,7 +142,7 @@ object SbtWsdl extends AutoPlugin {
       )
       .value,
     watchSources in Defaults.ConfigGlobal ++= (sources in wsdlToCode).value,
-    target in wsdlToCode := crossTarget.value / "wsdl" / Defaults.nameForSrc(configuration.value.name),
+    target in wsdlToCode := target.value / "wsdl" / Defaults.nameForSrc(configuration.value.name),
```

Comment:
Generated code moved to the target folder for support cross Scala version compilation.
Otherwise code generating to folder `target/scala-2.12` and then this code compiled to folder `target/scala-2.13/clasess`. It's a counterintuitively. 🤷‍♂ And breaks down some tests 😄